README.md

```diff
@@ -2,6 +2,160 @@
 
 Command-line interface for Jetfire/Transformations.
 
+## Usage
+
+### Authenticate
+
+To use `jetfire-cli`, the `JETFIRE_API_KEY` environment variable must be set to a valid API key for a service account which has access to Jetfire/Transformations.
+
+By default, `jetfire-cli` runs against the main CDF cluster (europe-west1-1).
+To use a different cluster, specify the `--cluster` parameter. Note that this is a global parameter, which must be specified before the subcommand.
+For example:
+
+```
+jetfire --cluster=greenfield <subcommand> [...args]
+```
+
+### Deploy transformations
+
+The primary purpose of `jetfire-cli` is to support continuous delivery, allowing you to manage transformations in a version control system.
+
+Transformations are described by YAML files, whose structure is described further below in this document.
+
+It is recommended to place these manifest files in their own directory, to avoid conflicts with other files.
+
+#### From the command line
+
+To deploy a set of transformations, use the `deploy` subcommand:
+
+```
+jetfire deploy <path>
+```
+
+The `<path>` argument should point to a directory containing YAML manifests.
+This directory is scanned recursively for `*.yml` and `*.yaml` files, so you can organize your transformations into separate subdirectories.
```

Comment:
We don't need this flexibility, I'd just require the user to use .yaml until we get a feature request with good arguments for allowing .yml. If we allow both, both will be used, and it's just annoying. No big deal, of course.

Reply:
I agree on a purely ideological level, the authors behind the yaml spec have publicly stated that `.yaml` is the officially recognized file extension, and having two file extensions for the same format is really annoying. But the use of `.yml` is so widespread that I'm leaning towards the pragmatic option here. For example, in the `cognitedata` organization, we have 2653 files with the `.yaml` extension, and 1105 files with the `.yml` extension. It's also important to consider the failure mode here. If you create a `.yml` file, it would just fail silently. No error message, your transformations is simply not deployed, and you may only notice after wondering why your data is not showing up. That's a really terrible user experience, and recognizing two file extensions seems like a small price to pay for avoiding that imo.

Reply:
Well, alright. :+1: